README.md

```diff
@@ -3,13 +3,58 @@ Welcome to `dlings`. This project aims to teach you how to program in D.
 If the name doesn't sound strange, it's because you might have heard of [rustlings](https://github.com/rust-lang/rustlings) or [ziglings](https://github.com/ratfactor/ziglings). In fact, this is **heavily** based on [rustlings](https://github.com/rust-lang/rustlings). If you're familiar with how [rustlings](https://github.com/rust-lang/rustlings) work, you will have no trouble getting started with `dlings`.
 
 # Getting started
+## Dependencies
+* [argparse](https://code.dlang.org/packages/argparse)
+* [fswatch](https://code.dlang.org/packages/fswatch)
+
+## Clone the repository
 ```sh
 git clone git@github.com:iK4tsu/dlings.git
 cd dlings
+```
+
+## Using DUB
+```sh
 dub build -b release
 ./dlings --help
 ```
 
+## Installing with Meson
+__Note:__ DUB prioritizes `dmd` when building dependencies. Since Meson is
+using DUB's local repository to find dependencies, you must ensure both
+Meson and DUB are using the same compiler.
+
+## Manually fetch and build dependencies
+```sh
+dub fetch <dependency>
+dub build <dependency>
+```
+
+__Note:__ You can choose another compiler with `--compiler`
+```sh
+dub build <dependency> --compiler=dmd
+```
+
+Assuming all needed dependencies are in your DUB's local repository, you can
+install `dlings` with:
```

Comment:
```suggestion

This will build the DUB dependencies and place then in your DUB's local
repository.

Now that we have all needed dependencies are in DUB's local repository,
you can install `dlings`:
```

Reply:
Fixed in 850b144577817e4a9dc1673b4b521f665dcc43c1